k8s/tools/block-aws/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:2-slim
```

Comment:
Have you tried this with python 3?

Reply:
it was supposed to be python3! Great catch!